src/NUnitFramework/framework/Api/DefaultTestAssemblyBuilder.cs

```diff
@@ -124,20 +124,25 @@ public ITest Build(string assemblyName, IDictionary<string, object> options)
 
         private TestSuite Build(Assembly assembly, string assemblyPath, IDictionary<string, object> options)
         {
-            TestSuite testAssembly = null;
-
             try
             {
-                if (options.ContainsKey(FrameworkPackageSettings.DefaultTestNamePattern))
-                    TestNameGenerator.DefaultTestNamePattern = options[FrameworkPackageSettings.DefaultTestNamePattern] as string;
+                var testNamePattern = GetValueOrDefault(options, FrameworkPackageSettings.DefaultTestNamePattern) as string;
+                if (testNamePattern != null) TestNameGenerator.DefaultTestNamePattern = testNamePattern;
```

Comment:
Please don't arbitrarily change code that isn't relevant to the thing you are working on based on personal stylistic preferences.